Makefile.build

```diff
@@ -758,14 +746,6 @@ plugins/micromega/sos_FORPACK:=
 plugins/micromega/sos_print_FORPACK:=
 plugins/micromega/csdpcert_FORPACK:=
 
-plugins/micromega/%.cmx: plugins/micromega/%.ml
-	$(SHOW)'OCAMLOPT  $<'
-	$(HIDE)$(OCAMLOPT) $(COND_OPTFLAGS) $(HACKMLI) $($(@:.cmx=_FORPACK)) -package unix,num -c $<
```

Comment:
Why is this part removed? `micromega` still depends on `num`.

Reply:
Indeed, it seems a better idea to keep it for now.